Student/Student/Submissions/SubmissionDetails/SubmissionDetailsPresenter.swift

```diff
@@ -84,7 +84,7 @@ class SubmissionDetailsPresenter: PageViewLoggerPresenterProtocol {
 
     func viewIsReady() {
         submissions.refresh(force: true)
-        assignment.refresh()
+        assignment.refresh(force: true)
```

Comment:
Force refresh to get the most current data on this screen since there is no pull to refresh.